Doc/library/fractions.rst

```diff
@@ -121,12 +121,11 @@ another rational number, or from a string.
 
    .. attribute:: numerator
 
-      Numerator of the Fraction in lowest term.
+      Numerator of the Fraction in lowest terms.
```

Comment:
In fact, we say that "The [Fraction](https://docs.python.org/3/library/fractions.html#fractions.Fraction) class inherits from the abstract base class [numbers.Rational](https://docs.python.org/3/library/numbers.html#numbers.Rational), and implements all of the methods and operations from that class."

So, we could move these numerator/denominator descriptions to the Rational class.

CC @AA-Turner

Reply:
That would be worse; it requires people to look in two places to find out what happens with `Fraction`. Happy to improve in both places, but we shouldn't remove from here.

Reply:
This pr continued in https://github.com/python/cpython/pull/136800.  I hope, I'll address review quickly;-)

> but we shouldn't remove from here.

I'm not sure if it worth verbosity.  I don't see e.g. where described real/imag properties for builtin types, except for the numbers module.

Reply:
@skirpichev should this PR (#122464) be closed?

A